SWLOR.Game.Server/Feature/GuiDefinition/ViewModel/AreaManagerViewModel.cs

```diff
@@ -0,0 +1,829 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using SWLOR.Game.Server.Core.NWNX;
+using SWLOR.Game.Server.Core.NWScript.Enum;
+using SWLOR.Game.Server.Core.NWScript.Enum.VisualEffect;
+using SWLOR.Game.Server.Service;
+using SWLOR.Game.Server.Service.GuiService;
+using SWLOR.Game.Server.Service.GuiService.Component;
+using SWLOR.Game.Server.Entity;
+using SWLOR.Game.Server.Service.DBService;
+
+namespace SWLOR.Game.Server.Feature.GuiDefinition.ViewModel
+{
+    public class AreaManagerViewModel: GuiViewModelBase<AreaManagerViewModel, GuiPayloadBase>
+    {
+        private readonly List<uint> _areas = new();
+        private readonly List<uint> _objects = new();
+        private readonly List<int> _objectAppearanceIds = new();
+        private readonly List<string> _objectAppearanceNames = new();
+
+        private static Dictionary<int, Dictionary<int, string>> _placeableAppearanceByPage { get; } = new();
```

Comment:
I don't think this needs to be a property. I'd get rid of the {get;} and treat it like a regular private field unless there's a reason for it.